mono/utils/mono-path.c

```diff
@@ -177,27 +177,129 @@ mono_path_resolve_symlinks (const char *path)
 #endif
 }
 
+#ifndef mono_host_win32
+#ifdef HOST_WIN32
+#define mono_host_win32 TRUE
```

Comment:
Just use `HOST_WIN32` around the code. There are only 3 uses of `mono_host_win32` so it will be easy I am sure.

Reply:
I changed to `HOST_WIN32`. The test deliberately does not use `ifdef`, in order to get win32 and non-win32 coverage, on win32 and non-win32 platforms; without affecting the performance of optimized mainline builds -- `HOST_WIN32` still being a constant `#define`, `0` or `1` there, but not `#ifdef`.